### ISSUE TITLE
Youtube still fights the new version 5.3

### ISSUE THREAD

**Issue by Badassious** [TheRealJoelmatic/RemoveAdblockThing#444]
**Script version**
What version is the script on eg 2.9 , 3.1 ect

**Describe the bug**

**Expected behavior**

**Screenshots**
If applicable, add screenshots or a video to help explain your problem.

**Desktop (please complete the following information):**

**Additional context**

**Comment by sunny10010** [comment]
Yes, there are some issue after upgrading to new version. In comparision to old version, Bypassing the youtube ads is apparently slow and do sometimes dump when it works.
if it could be changed back, please roll back to previous version. thanks!

**Comment by 3lawix9** [comment]
Yeah I hate that every video i have to press F5 3 times

**Comment by Badassious** [comment]
[3lawix9](https://github.com/3lawix9)

> Yeah I hate that every video i have to press F5 3 times

Watch again. That's no refresh, 3lawix9. It's ads trying to start.

**Comment by Badassious** [comment]
> I've updated the script to fix the detection. Please let me know if it works for u now. v5.4
> 
> also this a duplicate issue. (#439)

The videos play without interruption yet you can hear both sound of the current video and from the ad. I restarted the app and it still plays sound from ads.

**Comment by Tiropitaki1** [comment]
There's an acho of videos, not just only ads, but the video itself repeates the audio even at pause. Kinda like the first 10 seconds.
